.golangci.toml

```diff
@@ -71,40 +67,29 @@ disable-all = true
 enable = [
   "bodyclose",
   "depguard",
-  "dogsled",
-  "errcheck",
-  # "gochecknoinits",
-  "goconst",
```

Comment:
`goconst` got upgraded and now complains about dozens of strings here and there (usually logs or parts of the DB queries). It makes no sense to implement all its suggesions.